### PR TITLE
added comparison() function to functions file

### PR DESCRIPTION
**Pull request by ChrisTregaskis**
> in functions.js
> needs 2 parameters to compare
> must return true or false

### CODE REVIEW

**Review by alexstonelunde — APPROVED** [review]

**Review by ajhswan — COMMENTED** [review]
approved - Please can you wait for approval in writing for audit purposes. I'd don't want to have to tell you this again.....

**Pull request merged**
